src/contract/nodes.rs

```diff
@@ -366,7 +365,6 @@ pub struct Genesis {
     metadata: Metadata,
     owned_rights: OwnedRights,
     public_rights: PublicRights,
-    script: SimplicityScript,
```

Comment:
All scripts must be defined at schema level and specific nodes should not be able to override this, otherwise security model becomes unmanageble. If somebody needs different scripting, he has to create custom schema/do a subschema and than issue contract under it; and all nodes MUST use that validation rules (and not arbitrary be able to rewrite them)

Reply:
Ok. So that means smart contracting business logic have to be predecided at the time of asset creation?

corollary question: cant we use a bitcoin like stack based VM for our scripting purpose? If not what kind of language we will be using for the actual script code?